v1.1/select.md

```diff
@@ -528,6 +528,25 @@ LIMIT 5;
 +----+------------------+
 ~~~
 
+Alternatively, you can use `FETCH FIRST`.
+
+~~~ sql
+> SELECT id, name 
+FROM accounts 
+FETCH FIRST 5 ROWS ONLY; 
```

Comment:
I think we should put a warning alongside this if we're going to include an example of limit/fetch first that doesn't also include an `ORDER BY`, since the results are not going to be stable across invocations.

Reply:
I think it's sufficient to call out this compatibility-based syntax in the parameter list above. Let's remove this alternative example.